### DISCUSSION TITLE
Could you provide an upgrade path document ?

### DISCUSSION THREAD

**Discussion by ns-mglaske** [immich-app/immich#10762] [General]
Currently I'm on 1.98.2.  I typically update to major releases, and I suppose I'll just follow through the releases on the release page, but it would be super helpful for an upgrade path document.

eg as example:
1.98.2 -> 1.100.1 (along with things you might need to do)
1.100.1 -> 1.102.5
.. etc

**Comment by bo0tzz** [comment]
https://github.com/immich-app/immich/discussions/categories/announcements?discussions_q=category%3AAnnouncements+label%3Abreaking-change+sort%3Adate_created
